lib/slashdeploy/config.rb

```diff
@@ -0,0 +1,46 @@
+module SlashDeploy
+  # Config is a Ruby representation of the .slashdeploy.yml configuration file.
+  #
+  # Example
+  #
+  #   ---
+  #   environments:
+  #     production:
+  #       aliases:
+  #         - prod
+  #       continuous_delivery:
+  #         ref: refs/heads/master
+  #         required_contexts:
+  #           - ci/circleci
+  #     staging:
+  #       aliases:
+  #         - stage
+  class Config
+    include Virtus.model
```

Comment:
Just for my own understanding, why is this included in each method individually instead of just "at the top"?

Reply:
If a class is nested under another class, it doesn't inherit anything from the class above it. Generally when you nest classes like this, you're only doing it to namespace the class (e.g. `SlashDeploy::Config::Environment`).